test/compress/object.js

```diff
@@ -252,6 +252,13 @@ property_with_operator_value: {
             },
             *"*"() {
                 return "foobar";
+            },
+            "%": "foo",
+            get "%"() {
+                return "bar";
+            },
+            *"%"() {
+                return "foobar";
```

Comment:
You don't need any return statements here at all for the purpose of this test to make the `expect_exact` string shorter. Or you can just have 1; 2; 3; etc, if you want to easily visually cross reference the specific failing part of the test.